### PR TITLE
refactor(mobile): services and providers

### PR DESCRIPTION
**Pull request by alextran1502**
- Move all services to `services` directory at root level of `lib`
- Move all providers to `providers` directory at root level of `lib`, keep some providers grouped with logical level since there are many providers within that group

image

- Removed unused providers

### CODE REVIEW

**Comment by fyfrey** [comment]
Did you only move files around?
Could you summarize the changes made to discuss? Reviewing 200+ changed files... Takes too long :)

**Comment by alextran1502** [comment]
@fyfrey Done, update the description

**Review by fyfrey — APPROVED** [review]
Lovely, I like the restructuring. All services in one folder is perfect!
